app/models.py

```diff
@@ -762,6 +764,14 @@ def __init__(self, **kwargs):
     archived = db.Column(db.Boolean, nullable=False, default=False)
     hidden = db.Column(db.Boolean, nullable=False, default=False)
     subject = db.Column(db.Text)
+    postage = db.Column(db.String, nullable=True)
+    CheckConstraint("""
```

Comment:
Constraints defined in python are only relevant if we construct the database from python (which we don't do). It's good to have it here so we can refer to it without having to dig out migration scripts, but you'll need to also call alter table in the migration script

Reply:
ahh, cool, good to know, I will look how  this was done when postage was added to Notification 👍